app/views/layouts/_beta-banner.html.erb

```diff
@@ -0,0 +1,8 @@
+<div class="govuk-phase-banner">
+  <p class="govuk-phase-banner__content">
+    <strong class="govuk-tag govuk-phase-banner__content__tag <%= beta_tag_environment_class %>">
+      <%= beta_banner_environment_label %>
+    </strong>
+    <span class="govuk-phase-banner__text">This is a new service – your <%= bat_contact_mail_to "feedback", subject: "Publish teacher training courses feedback" %> will help us to improve it.</span>
```

Comment:
I think we can probably remove the feedback link. Even if we don't this is the wrong text.

Reply:
Removed.